### PR TITLE
Fixed issue when put max value transaction details are not presented

### PR DESCRIPTION
**Pull request by james-lin71**

### CODE REVIEW

File: src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts [hunk 1]
```diff
@@ -703,9 +700,19 @@ export class FormTransferTransactionTs extends FormTransactionBase {
...
             //@ts-ignore
             t.maxFee = UInt64.fromUint(maxFee);
+
+            if (t.mosaics.length) {
+                t.mosaics.map((m) => {
+                    const info = mosaicsInfo.find((i) => i.mosaicIdHex === m.id.toHex());
+                    const div = info ? info.divisibility : 0;
+                    const amount = m.amount.compact() / Math.pow(10, div);
+                    return new Mosaic(m.id, UInt64.fromUint(amount));
+                });
```
Comment (myxmaster) [inline src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts:714]: What is this good for? The result of that map is not used...

Comment (james-lin71) [reply src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts]: Refactored.

File: src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts [hunk 2]
```diff
@@ -330,16 +330,13 @@ export class FormTransferTransactionTs extends FormTransactionBase {
...
             .filter((attachment) => attachment.uid) // filter out null values
             .map(
                 (attachment): Mosaic => {
                     const amount = Number(attachment.amount.trim().replace(DECIMAL_SEPARATOR, '.'));
-                    const info = mosaicsInfo.find((i) => i.mosaicIdHex === attachment.mosaicHex);
-                    const div = info ? info.divisibility : 0;
-                    // - format amount to absolute
-                    return new Mosaic(new MosaicId(RawUInt64.fromHex(attachment.mosaicHex)), UInt64.fromUint(amount * Math.pow(10, div)));
+                    const mosaicId = new MosaicId(RawUInt64.fromHex(attachment.mosaicHex));
+                    return new Mosaic(mosaicId, UInt64.fromUint(amount));
```
Comment (bassemmagdy) [inline src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts:339]: this will let transactions have the wrong amount.

Comment (james-lin71) [reply src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts:339]: I resolved this.

**Review by bassemmagdy — APPROVED** [review]

**Pull request merged**